apps/src/Sound.js

```diff
@@ -339,22 +340,61 @@ Sound.prototype.getPlayableFile = function() {
   return false;
 };
 
+Sound.prototype.getPlayableBytes = function() {
```

Comment:
Would it make sense to move the functions that are specific to the speech sdk into their own file which imports Sounds.js and Sounds.js? IIRC, this file also gets used for a variety of other sound functions, so it may be good to keep it streamlined to the sections that are generic.